packages/@aws-cdk/aws-codepipeline-actions/lib/bitbucket/source-action.ts

```diff
@@ -0,0 +1,107 @@
+import * as codepipeline from '@aws-cdk/aws-codepipeline';
+import * as iam from '@aws-cdk/aws-iam';
+import { Construct } from '@aws-cdk/core';
+import { Action } from '../action';
+import { sourceArtifactBounds } from '../common';
+
+/**
+ * Construction properties for {@link BitBucketSourceAction}.
+ *
+ * @experimental
+ */
+export interface BitBucketSourceActionProps extends codepipeline.CommonAwsActionProps {
+  /**
+   * The output artifact that this action produces.
+   * Can be used as input for further pipeline actions.
+   */
+  readonly output: codepipeline.Artifact;
+
+  /**
+   * The ARN of the CodeStar Connection created in the AWS console
+   * that has permissions to access this BitBucket repository.
```

Comment:
Reference to AWS Docs on how to create the connection?

Reply:
Done.